src/commands/binder.ts

```diff
@@ -31,3 +38,22 @@ export function registerCommandSetThemeDay(
     setTheme(context, true);
   });
 }
+
+/**
+ * Creates and returns the Light Switch Status Bar Item
+ *
+ * Note: The returned StatusBarItem still needs to be pushed into vscode's subscriptions
+ */
+export function createLightSwitchStatusBarItem(): StatusBarItem {
+  const statusBarItem = window.createStatusBarItem(
+    StatusBarAlignment.Left,
+    100 // The higher the number the farther left it is on the status bar
+  );
+
+  statusBarItem.command = `lightSwitch.toggleThemess`;
```

Comment:
1. There's a typo here - it should be `toggleThemes` (Minus one `s`).
2. Can you load `lightSwitch` from `util/Constants`?